airflow/contrib/example_dags/example_gcp_automl_operators.py

```
@@ -0,0 +1,306 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that uses Google AutoML services.
+"""
+from typing import List, Dict
+import os
+from copy import deepcopy
+
+import airflow
+from airflow import models
+from airflow.contrib.hooks.gcp_automl_hook import get_object_id
+from airflow.contrib.operators.gcp_automl_operator import (
+    AutoMLTrainModelOperator,
+    AutoMLTablesListColumnSpecsOperator,
+    AutoMLPredictOperator,
+    AutoMLBatchPredictOperator,
+    AutoMLCreateDatasetOperator,
+    AutoMLTablesUpdateDatasetOperator,
+    AutoMLTablesListTableSpecsOperator,
+    AutoMLImportDataOperator,
+    AutoMLDeleteDatasetOperator,
+    AutoMLDatasetListOperator,
+    AutoMLDeployModelOperator,
+    AutoMLGetModelOperator,
+    AutoMLDeleteModelOperator,
+)
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "your-project-id")
+# For now only this location is supported
+GCP_AUTOML_LOCATION = os.environ.get("GCP_AUTOML_LOCATION", "us-central1")
+GCP_AUTOML_DATASET_BUCKET = os.environ.get("GCP_AUTOML_DATASET_BUCKET", "gs_source")
```

Is it perhaps safer to fail early when we we don't have these values? `gs_source` is not a safe default

---

This is only an example, I think we can assume that Airflow user will customise the DAG to fit his/her needs before using it.